pages/_app.tsx

```diff
@@ -7,36 +7,36 @@ import { AnchorsProvider } from "../components/util/Anchors";
 import { LanguageContext } from "../components/util/Contexts";
 import AuthPage from "../components/AuthPage";
 import fetcher from "../modules/fetcher";
-import homeIcon from "../assets/images/component-icons/side-nav/home-nav-icon.svg";
-import guidesIcon from "../assets/images/component-icons/side-nav/guides-nav-icon.svg";
-import conceptsIcon from "../assets/images/component-icons/side-nav/concepts-nav-icon.svg";
-import sdksToolsIcon from "../assets/images/component-icons/side-nav/sdks-tools-nav-icon.svg";
+import { ReactComponent as HomeIcon } from "../assets/images/component-icons/side-nav/home-nav-icon.svg";
+import { ReactComponent as GuidesIcon } from "../assets/images/component-icons/side-nav/guides-nav-icon.svg";
+import { ReactComponent as ConceptsIcon } from "../assets/images/component-icons/side-nav/concepts-nav-icon.svg";
+import { ReactComponent as SdksToolsIcon } from "../assets/images/component-icons/side-nav/sdks-tools-nav-icon.svg";
```

Comment:
Using inline SVGs so we can change the fill color when `.active` class is applied